tools/phpunit/phpunit.xml

```diff
@@ -9,6 +9,10 @@
          stopOnFailure="false"
          syntaxCheck="false"
          bootstrap="./../../vendor/autoload.php">
+    <php>
+        <!-- During testing we want all modules enabled -->
```

Comment:
Actually, we don't :smile: 

How could we test then that `SimpleSAML\Module::isModuleEnabled()` fails appropriately if all modules are enabled?

I think we need an alternative way to do this, using the `module.enable` configuration option that we have already.